### PR TITLE
Stop JDBC setting resource.name to db statement

### PR DESCRIPTION
**Pull request by sm7x**

### CODE REVIEW

File: dd-java-agent/instrumentation/hibernate/core-3.3/src/test/groovy/SessionTest.groovy [hunk 1]
```diff
@@ -83,14 +85,14 @@ class SessionTest extends AbstractHibernateTest {
     }
 
     where:
-    testName  | methodName | resource | sessionImplementations                    | sessionMethodTest
-    "lock"    | "lock"     | "Value"  | [sessionBuilder]                          | { sesh, val ->
+    testName  | methodName | resource | sessionImplementations                      | sessionMethodTest
```
Comment (rmfitzpatrick) [inline dd-java-agent/instrumentation/hibernate/core-3.3/src/test/groovy/SessionTest.groovy]: nit: please avoid all unnecessary whitespace changes unless they improve readability in documentation.  I don't think columns need to line up to be a valid data table.

Comment (sm7x) [reply dd-java-agent/instrumentation/hibernate/core-3.3/src/test/groovy/SessionTest.groovy]: I found it easier to read/grasp with the columns aligned, that's why I did that.
